www/components/Nav/Annoucement.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { IconArrowRight } from '@supabase/ui'
+
+import data from '~/data/Annoucement.json';
+
+type A = {
+  show: boolean;
+  text: string;
+  link: {
+    text: string;
+    url: string;
+  }
+}
+
+const a: A = data;
+
+const Annoucement = () => {
+  if (!a.show) return (<></>)
+  return (
+    <div className="bg-brand-700 dark:bg-gray-700">
+      <div className="flex flex-col items-center justify-center py-2 mx-auto space-x-2 text-white sm:flex-row lg:container lg:px-16 xl:px-20">
+        <span>{a.text}</span>
+        <a
```

Comment:
would prefer to see @supabase/ui button used if it looks good. leave it up to you though 👍

Reply:
I retried but it doesn't look good imo

Reply:
👍